src/ui/components/Comments/CommentCardsList.tsx

```diff
@@ -11,57 +14,80 @@ import useAuth0 from "ui/utils/useAuth0";
 import styles from "./CommentCardsList.module.css";
 
 export default function CommentCardsList() {
+  const { currentUserInfo } = useContext(SessionContext);
+
   const recordingId = hooks.useGetRecordingId();
   const { comments, loading } = hooks.useGetComments(recordingId);
   const recording = hooks.useGetRecording(recordingId);
   const auth = useAuth0();
 
+  const { filter, sortBy } = useUserCommentPreferences();
+
   const { isAuthenticated } = useAuth0();
 
   const displayedComments = useMemo(() => {
-    const clonedComments: Comment[] = [...comments];
+    const filteredComments =
```

Comment:
:speech_balloon: my one style nit here is that I'm _really_ not a fan of nested ternaries :)

Reply:
Yeah that's fair. I'll rewrite this bit.